apps/allocations/app/components/Form/Form.js

```diff
@@ -15,6 +22,7 @@ const Form = ({ children, onSubmit, submitText, heading, subHeading }) => {
         mode="strong"
         wide
         onClick={onSubmit}
+        disabled={disabled}
```

Comment:
In a separate, follow-up PR, I'd love to see us get rid of the `onClick` on this button, and move that logic to a `<form onSubmit>` instead.